app/models/concerns/with_qualifications.rb

```diff
@@ -45,7 +45,7 @@ module WithQualifications
 
     # This field may seem like an unnecessary overhead when there is already a
     # database-backed `qualification` field. However it's misleading, from the
-    # point of view of the teacher training domain, to think of 'PGCE with QTS'
+    # point of view of the teacher training domain, to think of 'QTS with PGCE'
```

Comment:
On line 68 `when 'pgce_with_qts' then 'Postgraduate certificate in education (PGCE) with qualified teacher status (QTS)'` should we be updating this wording too?

Reply:
I pushed the fix. Is that the only place you found?

Reply:
This is the only place that I saw - I didn't look very far to be honest